mmpose/apis/inference.py

```diff
@@ -52,6 +52,8 @@ def _xyxy2xywh(bbox_xyxy):
         np.ndarray: Bounding boxes (with scores),
           shaped (n, 4) or (n, 5). (left, top, width, height, [score])
     """
+    if len(bbox_xyxy) == 0:
+        return []
```

Comment:
but the docstring says the return type is  np.ndarray

Reply:
this will change the returned type from np array to list, so it is not ideal.

if the purpose of the change is to speed up things, then you can skip the two lines after copy:

```
if len(bbox_xyxy) > 0:
    bbox_xywh[:, 2] = bbox_xywh[:, 2] - bbox_xywh[:, 0] + 1
    bbox_xywh[:, 3] = bbox_xywh[:, 3] - bbox_xywh[:, 1] + 1
```